### DISCUSSION TITLE
Using cobra to auto generate cli docs

### DISCUSSION THREAD

**Discussion by rsjethani** [ioki-mobility/go-outline#44] [Ideas]
See: https://github.com/spf13/cobra/blob/main/site/content/docgen/_index.md

cc @StefMa  @zadjadr

**Comment by StefMa** [comment]
Boha 🤯 this is nice!
Let us try to make use of this and add it (for now) to the Readme. Once we have a lot of stuff, let us create a github page 👍

I mean we don't have any pressure with doing that. Especially because we only have a handful of commands right now 😂 however I think this is fun!

**Comment by StefMa** [comment]
Created an issue it of this to try it out:
https://github.com/ioki-mobility/go-outline/issues/46
